code/_globalvars/lists/vox.dm

```diff
@@ -0,0 +1,15 @@
+GLOBAL_LIST_INIT(available_vox_voices, list(
+	"GLaDOS",
+	//"Wheatley",
```

Comment:
Sad

Reply:
maybe when he comes back

Reply:
But he is already in space! He should have more of a right than anyone else in this list.